src/WPF/ArcGISRuntime.WPF.Viewer/Resources/SyntaxHighlighting/highlight.css

```diff
@@ -0,0 +1,104 @@
+.xml .hljs-tag .hljs-attribute, .html .hljs-tag .hljs-attribute {
+    color: #aa0d91;
+}
+/*
+ XCode style (c) Angel Garcia <angelgarcia.mail@gmail.com>
```

Comment:
I recommend using a theme that is similar to the one used by Visual Studio, because that will be most comfortable/familiar for our developers. It looks like there is one available [here](https://highlightjs.org/static/demo/styles/vs.css)

Reply:
I am currently using the one from the documentation website. Purpose is to have consistency with the website. What do you think is better? I tired your css that you linked and it looks nearly identical.

Reply:
The differences are very minor and given that it is consistent with the website, I think that's totally fine.